models/refugee.py

```diff
@@ -0,0 +1,105 @@
+from datetime import date, datetime
+
+class Refugee:
+    """
+    A class to represent a refugee family.
+    """
+    def __init__(self,
+                 name: str,
+                 camp: str,
+                 medicalCondition: str,
+                 numOfFamilyMember: int,
+                 dateOfClosing=None):
+
+        """
+        Initialise a new refugee.
+        :param name of refugee family
+        :param number of family member
+        :param camp of the refugee family locating at
+        :param medical condition of the refugee
+        :param start date of refugee family
+        :param closing date of refugee family
+        """
+
+        self.name = name
+        self.numOfFamilyMember = self.__checkNumOfFamilyMember(numOfFamilyMember)
+        self.camp = camp
+        self.medicalCondition = medicalCondition
+        self.dateOfCreation = datetime.today().date()
+        self.dateOfClosing = self.__checkDateOfClosing(dateOfClosing)
+
+    def formatName(self, name):
+        """
+        format name to a new name
+        """
+        self.name = name
+        #error handling if invalid name is inputed
```

Comment:
What does this method do? does it rename the refugee?
If so maybe naming it `rename()` would be more clear. 
Also, since the attribute `name` is public now, it doesn't really need a method to change its value. 
If the motivation is for implementing name validation in the future, we should make the `name` attribute private.